src/components/layout/languageDropdown/LanguageDropdown.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Dropdown } from 'reactstrap';
+
+import { shallowWithIntl } from '../../../utils/testUtils';
+import LanguageDropdown from './LanguageDropdown';
+
+describe('test', () => {
```

Comment:
`'test'` might not be the most suitable name here, maybe change to something more descriptive.

Reply:
Oops 🤦‍♂